src/test/scala/util/PostgresFixture.scala

```diff
@@ -13,36 +10,36 @@ import scala.util.Random
   * Created by hsslbch on 9/9/16.
   */
 trait PostgresFixture extends BaseFixture with BeforeAndAfterAll { self: Suite =>
-
-  override def afterAll(): Unit = {
-    super.afterAll()
-    if (manageConnectionIsOpened) {
-      manageConnection.close()
-    }
+  import ExecutionContext.Implicits.global
+
+  /** Execute a test against all supported versions of Postgres in parallel. */
+  def withPostgres(fixtureName: String)
+                   (testCode: BaseConnection => Future[Assertion]): Future[Assertion] = {
+    Future.traverse(PostgresFixture.PostgresImages) { image =>
+      val dbName = "test_" + Random.alphanumeric.take(6).mkString.toLowerCase()
+      PostgresContainer.withPostgresContainer(image) { connection =>
+        connection.createStatement().execute(s"CREATE DATABASE $dbName;")
+        connection.createStatement().execute(Resource.read(s"/fixtures/$fixtureName.sql"))
+
+        testCode(connection)
+      }
+    }.map(reduceAssertions)
   }
 
-  private val config = ConfigFactory.load().getConfig("postgres")
-  private val url = config.getString("url")
-  private val username = config.getString("username")
-  private val password = config.getString("password")
```

Comment:
If config is not taken into account anymore, could you drop it then, please? (I mean the whole `application.conf` file)